steam-tweaks.yaml

```diff
@@ -1,1476 +0,0 @@
-# DO NOT EDIT THIS FILE
```

Comment:
we should keep this file for a time for backwards compatibility

Reply:
OK. Until we migrate completely. But think about the static file that is distributed, The app will work but won't be updated.